main.go

```
@@ -51,203 +45,215 @@ func (pd *PopData) SaveRpzSerial() error {
+}
+
+func loadConfigFiles(stderr io.Writer) (string, error) {
+	viper.Reset()
+	viper.SetConfigFile(tapir.DefaultPopCfgFile)
+	viper.AutomaticEnv()
+
+	if err := viper.ReadInConfig(); err != nil {
+		return "", fmt.Errorf("could not load config %s: %w", tapir.DefaultPopCfgFile, err)
+	}
+	cfgFileUsed := viper.ConfigFileUsed()
+	fmt.Fprintln(stderr, "Using config file:", cfgFileUsed)
```

golangci-lint (2.12.1)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)